tests/test_density_LocalDensity.py

```diff
@@ -42,22 +41,23 @@ def test_density(self):
         """Test that LocalDensity computes the correct density at each point"""
 
         r_max = self.r_max + 0.5 * self.diameter
-        test_set = util.make_raw_query_nlist_test_set(
-            self.box, self.pos, self.pos, "ball", r_max, 0, True
-        )
-        for nq, neighbors in test_set:
-            self.ld.compute(nq, neighbors=neighbors)
 
-            # Test attribute access after calling compute
-            self.ld.density
-            self.ld.num_neighbors
-            self.ld.box
+        # This test is slow, so we only test AABBQuery.
+        nq = freud.locality.AABBQuery(self.box, self.pos)
+        neighbors = {"mode": "ball", "r_max": r_max, "exclude_ii": True}
+
+        self.ld.compute(nq, neighbors=neighbors)
+
+        # Test attribute access after calling compute
+        self.ld.density
+        self.ld.num_neighbors
+        self.ld.box
 
-            assert self.ld.box == freud.box.Box.cube(10)
+        assert self.ld.box == freud.box.Box.cube(10)
 
-            npt.assert_array_less(np.fabs(self.ld.density - 10.0), 1.5)
+        npt.assert_array_less(np.fabs(self.ld.density - 10.0), 1.5)
 
-            npt.assert_array_less(np.fabs(self.ld.num_neighbors - 1130.973355292), 200)
+        npt.assert_array_less(np.fabs(self.ld.num_neighbors - 1130.973355292), 200)
```

Comment:
Can we add a comment for the magic numbers in this test? I have no idea where they come from.

Reply:
I would love to, but I don't know either. I'd assume from the use of `np.fabs` that this is a regression test Harper wrote way back in the day.